phpunit.xml.dist

```diff
@@ -9,7 +9,8 @@
 		 colors="true"
 		 failOnRisky="true"
 		 failOnWarning="true"
-		 verbose="true">
+		 verbose="true"
+		 ignoreDeprecatedCodeUnits="true">
 
 	<coverage processUncoveredFiles="true">
```

Comment:
```suggestion
		 verbose="true">

	<coverage processUncoveredFiles="true" ignoreDeprecatedCodeUnits="true">
```
This should be on the `coverage` element per phpunit's XSD

Reply:
updated